docs/how-to/installation/aws.txt

```diff
@@ -418,9 +404,9 @@ Agent
 
 Master Node
 ~~~~~~~~~~~
-The master node should be deployed on an EC2 instance supporting >= 2 CPUs (Intel Broadwell or later), 4GB of RAM,
+The master node should be deployed on an EC2 instance with at least 2 CPUs (Intel Broadwell or later), 4GB of RAM,
```

Comment:
blocking: I definitely could be reading this wrong, but I think this is too low. I did find discussion here, though not sure if this is a special case or something https://determined-ai.slack.com/archives/CUJ0T4U01/p1586523555017600

Reply:
Looks like we list higher requirements elsewhere, so I'll bump these. Good eye!